plugins/joe_sandbox/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-jbxapi==2.9.5
+jbxapi==3.21.0
```

Comment:
Could you please update to the latest one `3.22.0` or there is a cause of having `3.21.0`? 😄

Reply:
I followed along with the customers update but that was back in November so I can update this now :)